web/cypress/test/extra-config.yaml

```diff
@@ -0,0 +1 @@
+FEATURE_AUTO_PRUNE: true
```

Comment:
We can enable it at some point, just leaving it disabled for now since it's still in development and I don't want to effect any other development efforts.